test/server/data/test-suites/regression-gh-599/testfile.js

```diff
@@ -0,0 +1,28 @@
+import { APIError } from '../../../../../lib/errors/runtime';
+
+fixture `Test`
+    .page `http://example.com`;
+
+var obj = {
+    method1: () => {
+        var err = new APIError('method1');
+
+        // NOTE: to preserve callsite
+        err.isTestCafeError = true;
```

Comment:
It will preserver callsite for `APIError` anyway: https://github.com/DevExpress/testcafe/blob/master/src/errors/process-test-fn-error.js#L30

Assigning this flag to `APIError` is just incorrect